src/web/server/app/controllers/notif.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response } from 'express';
+import * as notifServices from '../services/notif';
+
+export const retrieveNotifications = async (req: Request, res: Response) => {
+  const result = await notifServices.retrieveNotif();
+  if (result === undefined) {
+    res.status(400).send('No notifications found');
+  } else {
+    res.status(200).json(result);

```

Comment:
This will be a json of a ResultSet which is not what you want but I'm guessing it's just placeholder

Reply:
Well I plan on parsing the json from the frontend to get the necessary data to display each notifications.